code/game/gamemodes/clock_cult/clock_items/clockwork_proselytizer.dm

```diff
@@ -298,28 +158,4 @@
 		return FALSE
 	if(repair_values["amount_to_heal"] <= 0) //nothing to heal!
 		return FALSE
-	repair_values["healing_for_cycle"] = min(repair_values["amount_to_heal"], PROSELYTIZER_REPAIR_PER_TICK) //modify the healing for this cycle
-	repair_values["power_required"] = round(repair_values["healing_for_cycle"]*MIN_CLOCKCULT_POWER, MIN_CLOCKCULT_POWER) //and get the power cost from that
-	if(!can_use_power(RATVAR_POWER_CHECK) && !can_use_power(repair_values["power_required"]))
-		if(!silent)
-			to_chat(user, "<span class='warning'>You need at least <b>[repair_values["power_required"]]W</b> power to start repairin[target == user ? "g yourself" : "g [target]"], and at least \
-			<b>[round(repair_values["amount_to_heal"]*MIN_CLOCKCULT_POWER, MIN_CLOCKCULT_POWER)]W</b> to fully repair [target == user ? "yourself" : "[target.p_them()]"]!</span>")
```

Comment:
you do actually need this chunk, and to update it to use wisdom, as otherwise the heal amount won't get set.